### PR TITLE
enforce linting / formatting standards (#80)

### PR DESCRIPTION
**Pull request by abdrahmanES1**

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -1,8 +1,28 @@
...
+    "^@/components/ui/(.*)$",
+    "^@/components/(.*)$",
+    "^@/registry/(.*)$",
+    "^@/styles/(.*)$",
+    "^@/app/(.*)$",
+    "",
+    "^[./]"
+  ],
+  "plugins": [
+    "@ianvs/prettier-plugin-sort-imports"
```
Comment (safethecode) [inline .prettierrc:26]: typo 🥲
```suggestion
    "@trivago/prettier-plugin-sort-imports"
```

**Review by safethecode — CHANGES_REQUESTED** [review]

**Comment by safethecode** [comment]
@dillionverma 
I think the Pull Request unit is too large. We need to verify if it working

**Comment by dillionverma** [comment]
agreed I would prefer smaller PRs, but this gave me a lot of ideas of what to potentially try and how to setup prettier / linting

**Comment by alamenai** [comment]
HI @abdrahmanES1 ,

If your changes are independent, please could you [cherry-pick ](https://git-scm.com/docs/git-cherry-pick) your commits and open pull requests with small changes.

I know the files changes are affected by the linter but a detailed description can be very helpful for the maintainers.

@dillionverma , what do you think about changing the target branch to `dev` and verify the changes through code spaces.

**Pull request closed without merging**